lib/openhab/dsl/items/timed_command.rb

```diff
@@ -197,7 +201,10 @@ def initialize(timed_command_details)
             self.name = "Cancel implicit timer for #{timed_command_details.item.name}"
             self.triggers = [Rules::RuleTriggers.trigger(
               type: Rules::Triggers::Changed::ITEM_STATE_CHANGE,
-              config: { "itemName" => timed_command_details.item.name }
+              config: {
+                "itemName" => timed_command_details.item.name,
+                "previousState" => timed_command_details.command.to_s
```

Comment:
you should run this through `timed_command_details.item.format_command(command).to_s`.

Reply:
Good catch!